src/hotspot/cpu/x86/vm_version_x86.hpp

```diff
@@ -703,6 +703,11 @@ class VM_Version : public Abstract_VM_Version {
   static bool supports_cet_ss()       { return (_features & CPU_CET_SS) != 0; }
   static bool supports_cet_ibt()      { return (_features & CPU_CET_IBT) != 0; }
 
+  //
+  // Feature identification not affected by VM flags
+  //
+  static bool cpu_supports_evex()         { return (_cpu_features & CPU_AVX512F) != 0; }
```

Comment:
Indenting is a bit off here. I think the opening brace should be aligned with the others on the top.